src/layouts/ArtObjectPage/ArtObjectPage.jsx

```diff
@@ -82,7 +82,7 @@ class ArtObjectPage extends Component {
     const pageTitle = `${this.props.artObject.culture || this.props.artObject.people} - ${this.props.artObject.title}`;
 
     return (
-      <div className="app">
+      <div className="app app-art-object-page">
```

Comment:
I started this paradigm of using a class called app-* to name the page and nest the styles.